src/styles/blocks/place.scss

```diff
@@ -0,0 +1,49 @@
+.place {
+
+  &__photo-container {
+    position: relative;
+    box-sizing: border-box;
+    width: 100%;
+    max-height: 256px;
+    overflow: hidden;
+
+    &::after {
+      content: "";
+      display: block;
+      padding-bottom: 100%;
+    }
+
+    @include onDesktop {
+      height: 256px;
+      padding-bottom: 0;
+    }
+  }
+
+  &__photo {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    border-radius: 8px;
+    object-fit: cover;
+    object-position: center;
+    transition: width 0.3s linear 0.3s;
+
+    &:hover {
+      width: 110%;
+    }
```

Comment:
interesting solution, you can also try transform: scale() , maybe you would like it )